### PR TITLE
Edited the ovirt-image-template role.

### PR DESCRIPTION
**Pull request by bgraveno**
Hi @machacekondra,

Can you please review and merge this request?

Kind regards,

### CODE REVIEW

File: roles/ovirt-image-template/README.md [hunk 1]
```diff
@@ -1,33 +1,33 @@
...
-| template_memory    | 2GiB                  | Amount of memory of template |
-| template_cpu       | 1                     | Number of CPU of template  |
-| template_disk_storage | UNDEF              | Name of storage domain where disk should be created. If not specified the storage domain is choosen automatically |
-| template_disk_size | 10GiB                 | Size of the template disk  |
-| template_disk_format | cow                 | Format of the template disk  |
-| template_disk_interface | virtio           | Interface of the template disk |
-| template_timeout   | 600                   | Timeout to wait for the template to be created |
-| template_nics      | {name: nic1, profile_name: ovirtmgmt, interface: virtio} | List of dictionarie specifing the nics of template |
+| qcow_url           | UNDEF                 | The URL of the QCOW2 image. |
+| image_path         | /tmp/ovirt_image_data | Path where QCOW2 image must be downloaded. |
```
Comment (mwperina) [inline roles/ovirt-image-template/README.md]: When role is executed, image will be downloaded from qcow_url and stored locally into image_path. So please change the description to: 'Path where QCOW2 image will be downloaded.'

File: roles/ovirt-image-template/README.md [hunk 2]
```diff
@@ -1,33 +1,33 @@
...
+| template_cluster   | Default               | Name of the cluster where template must be created. |
+| template_name      | mytemplate            | Name of the template. |
+| template_memory    | 2GiB                  | Amount of memory assigned to the template. |
+| template_cpu       | 1                     | Number of CPUs assigned to the template.  |
+| template_disk_storage | UNDEF              | Name of the storage domain where the disk must be created. If not specified, the storage domain is selected automatically. |
```
Comment (mwperina) [inline roles/ovirt-image-template/README.md]: I think it should be mentioned that this is data storage domain and not export storage domain.

**Review by mwperina — CHANGES_REQUESTED** [review]

File: roles/ovirt-image-template/README.md (hunk 1 above)
Comment (bgraveno) [reply roles/ovirt-image-template/README.md]: Thanks, updated this to, "Path where the QCOW2 image will be downloaded to."

File: roles/ovirt-image-template/README.md (hunk 2 above)
Comment (bgraveno) [reply roles/ovirt-image-template/README.md]: Thanks, I've updated this to specifically mention "data storage domain".

**Review by machacekondra — APPROVED** [review]

**Pull request merged**